node/src/main/scala/coop/rchain/node/node.scala

```diff
@@ -181,33 +180,35 @@ class NodeRuntime(conf: Conf)(implicit scheduler: Scheduler) {
   implicit val nodeDiscoveryEffect: NodeDiscovery[Task] =
     new KademliaNodeDiscovery[Task](src, defaultTimeout)
 
+  if (Files.notExists(casperPersistencePath)) Files.createDirectories(casperPersistencePath)
```

Comment:
I would move this directory existence check into the `LMDBBlockStore.create` method itself.